### ISSUE TITLE
always run into problem running examples

### ISSUE THREAD

**Issue by sperfu** [flukeskywalker/highway-networks#2]
hi,I used your caffe model to run the highway network example mnnist classification

train_mnist.sh

but I ran into such problem :
I0707 22:55:59.848199 91625 layer_factory.hpp:74] Creating layer mnist
I0707 22:55:59.848260 91625 net.cpp:87] Creating Layer mnist
I0707 22:55:59.848281 91625 net.cpp:345] mnist -> data
I0707 22:55:59.848331 91625 net.cpp:345] mnist -> label
I0707 22:55:59.848356 91625 net.cpp:116] Setting up mnist
I0707 22:55:59.848512 91625 db.cpp:34] Opened lmdb /home/penglab/highway-networks/examples/highways/mnist-10layers/mnist_train_lmdb
I0707 22:55:59.848590 91625 data_layer.cpp:67] output data size: 100,1,28,28
I0707 22:55:59.848933 91625 net.cpp:123] Top shape: 100 1 28 28 (78400)
I0707 22:55:59.848994 91625 net.cpp:123] Top shape: 100 (100)
I0707 22:55:59.849011 91625 net.cpp:154] With 0 trainable parameters
I0707 22:55:59.849025 91625 layer_factory.hpp:74] Creating layer drop1
I0707 22:55:59.849047 91625 net.cpp:87] Creating Layer drop1
I0707 22:55:59.849063 91625 net.cpp:387] drop1 <- data
I0707 22:55:59.849090 91625 net.cpp:345] drop1 -> drop1
I0707 22:55:59.849112 91625 net.cpp:116] Setting up drop1
I0707 22:55:59.849140 91625 net.cpp:123] Top shape: 100 1 28 28 (78400)
I0707 22:55:59.849151 91625 net.cpp:154] With 0 trainable parameters
I0707 22:55:59.849161 91625 layer_factory.hpp:74] Creating layer s1-conv1
I0707 22:55:59.849189 91625 net.cpp:87] Creating Layer s1-conv1
I0707 22:55:59.849202 91625 net.cpp:387] s1-conv1 <- drop1
I0707 22:55:59.849217 91625 net.cpp:345] s1-conv1 -> s1-conv1
I0707 22:55:59.849237 91625 net.cpp:116] Setting up s1-conv1
I0707 22:55:59.849376 91625 net.cpp:123] Top shape: 100 32 24 24 (1843200)
I0707 22:55:59.849405 91625 net.cpp:154] With 832 trainable parameters
I0707 22:55:59.849416 91625 layer_factory.hpp:74] Creating layer relu
I0707 22:55:59.849433 91625 net.cpp:87] Creating Layer relu
I0707 22:55:59.849445 91625 net.cpp:387] relu <- s1-conv1
I0707 22:55:59.849457 91625 net.cpp:334] relu -> s1-conv1 (in-place)
I0707 22:55:59.849472 91625 net.cpp:116] Setting up relu
I0707 22:55:59.849484 91625 net.cpp:123] Top shape: 100 32 24 24 (1843200)
I0707 22:55:59.849494 91625 net.cpp:154] With 0 trainable parameters
I0707 22:55:59.849503 91625 layer_factory.hpp:74] Creating layer s1-conv2
F0707 22:55:59.849546 91625 layer_factory.hpp:77] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: CuDNNHighway (known types: AbsVal, Accuracy, ArgMax, BNLL, Concat, ContrastiveLoss, Convolution, Data, Deconvolution, Dropout, DummyData, Eltwise, EuclideanLoss, Exp, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, ReLU, Reshape, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, WindowData)
**\* Check failure stack trace: ***
    @     0x7f684da36daa  (unknown)
    @     0x7f684da36ce4  (unknown)
    @     0x7f684da366e6  (unknown)
    @     0x7f684da39687  (unknown)
    @     0x7f684de21af5  caffe::LayerRegistry<>::CreateLayer()
    @     0x7f684de26c39  caffe::Net<>::Init()
    @     0x7f684de28ab2  caffe::Net<>::Net()
    @     0x7f684de34cc0  caffe::Solver<>::InitTrainNet()
    @     0x7f684de35dc3  caffe::Solver<>::Init()
    @     0x7f684de35f96  caffe::Solver<>::Solver()
    @           0x40c608  caffe::GetSolver<>()
    @           0x406463  train()
    @           0x404991  main
    @     0x7f684cf47ec5  (unknown)
    @           0x404f3d  (unknown)
    @              (nil)  (unknown)
./examples/highways/mnist-10layers/train_mnist.sh: 行 5:

I successfully buit the caffe and pycaffe...but still ran into such a problem...
please give me some solution....thanks
